Draggable: Add Getting Started (#8276)
DevExpress/devextreme-documentation: concepts/05 UI Components/Draggable/05 Getting Started with Draggable/00 Getting Started with Draggable.md

#include tutorials-intro-installationnote

This tutorial guides you through the following steps: 

- Add a Draggable to a page.
- Configure Draggable options.

> - Handle events (specific to Draggable, as well as common events).

Reviewer: ```suggestion
- Handle common events and events specific to Draggable.
```
